python/ray/train/constants.py

```diff
@@ -79,3 +79,9 @@
 # By default these will be filtered out from ``session.report()``.
 # See ``TrainingCallback._preprocess_results`` for more details.
 ALL_RESERVED_KEYS = {PYTORCH_PROFILER_KEY}
+
+# Default NCCL_SOCKET_IFNAME.
+# Use ethernet when possible.
+# NCCL_SOCKET_IFNAME does a prefix match so "ens3" or "ens5" will match with
+# "en".
+DEFAULT_NCCL_SOCKET_IFNAME = "en,eth,bond"
```

Comment:
Double checking my understanding, does order matter here? Or does it matter what the order of `ifconfig` outputs?

Reply:
The first is prefered (first anything starting in en, then eth, then bond)